roles/arteria-sequencing-report-ws/templates/seqreport_app.config.j2

```diff
@@ -27,7 +27,8 @@ nextflow_config:
     # - ${DEFAULT:current_year}
     parameters:
         run_folder: ${DEFAULT:runfolder_path}
-        config_dir: {{ arteria_service_config_root }}
+        project: {{ uppmax_project }}
+        config_dir: {{ arteria_service_src_root }}/seqreports/config/tool_config/
```

Comment:
`arteria_service_src_root` is undefined, as far as I can tell. Shouldn't it be `{{ arteria_service_config_root }}/seqreports/config/tool_config/`?

Reply:
Ah, crap, it should be `{{ arteria_service_src_path }}`, gonna fix that, great catch!